drivers/crypto/crypto_mtls_shim.c

```diff
@@ -31,22 +31,36 @@
 #error "You need to define MBEDTLS_MEMORY_BUFFER_ALLOC_C"
 #endif /* MBEDTLS_MEMORY_BUFFER_ALLOC_C */
 
-#include <mbedtls/ccm.h>
 #include <mbedtls/aes.h>
+#include <mbedtls/ccm.h>
+
+#if defined(CONFIG_MBEDTLS_ENABLE_HMAC_DRBG)
 #include <mbedtls/hmac_drbg.h>
+#endif
+
+#if defined(CONFIG_MBEDTLS_ENABLE_CMAC)
 #include <mbedtls/cmac.h>
+#include <mbedtls/md.h>
+#endif
+
+#if defined(CONFIG_MBEDTLS_ENABLE_ECC_DH)
 #include <mbedtls/ecp.h>
 #include <mbedtls/bignum.h>
 #include <mbedtls/ecdh.h>
+#endif
 
 #define MTLS_SUPPORT (CAP_RAW_KEY | CAP_SEPARATE_IO_BUFS | CAP_SYNC_OPS)
 
 struct mtls_shim_session {
 	enum cipher_algo algo;
 	union {
 		mbedtls_ccm_context mtls;
```

Comment:
Since mtls is not covered under #ifdef like others, does it mean that aes-ccm is always enabled in the shim?